README.md

```diff
@@ -1,6 +1,7 @@
-# gitbase [![GitHub version](https://badge.fury.io/gh/src-d%2Fgitbase.svg)](https://github.com/mcuadros/ofelia/releases) [![Build Status](https://travis-ci.org/src-d/gitbase.svg?branch=master)](https://travis-ci.org/src-d/gitbase) [![codecov](https://codecov.io/gh/src-d/gitbase/branch/master/graph/badge.svg)](https://codecov.io/gh/src-d/gitbase) [![GoDoc](https://godoc.org/gopkg.in/src-d/gitbase.v0?status.svg)](https://godoc.org/gopkg.in/src-d/gitbase.v0) [![Go Report Card](https://goreportcard.com/badge/github.com/src-d/gitbase)](https://goreportcard.com/report/github.com/src-d/gitbase)
+![gitbase logo](https://cdn.rawgit.com/src-d/artwork/efe2dd7d/gitbase/files/gitbase-github-readme-header.png)
```

Comment:
## On the code

I'd say it's good practice to have the logo file also in the repository, so that:

1. You don't risk getting broken links if anything goes wrong with the server, domain, URL, etc., particularly over time.
1.a. As a bonus, you can always appreciate the logo when working offline.
2. You have version control of the logotype itself.

## On the logo + header/hero image itself

Pros:

- Plain looks looks very nice.
- Pairs well with the gitbase playground.

Cons:

- Missing references to git and to (data)base, that are the key elements of the project.
- The blue of the project description is too light and provides low contrast (particularly given the patterned background—better to go for the dark blue from the illustration outline).

Reply:
Thanks for the feedback Marcelo :)

> I'd say it's good practice to have the logo file also in the repository, so that:

I just followed the go-git README example, but it makes sense. I'll add it to the repo.

> Missing references to git and to (data)base, that are the key elements of the project.

Logos don't literally have to be literal. If you literally want it literal, I'll iterate. 

> The blue of the project description is too light and provides low contrast (particularly given the patterned background—better to go for the dark blue from the illustration outline).

Agree, will do.